docs/apis-clients/community-clients/quarkus.md

```diff
@@ -0,0 +1,15 @@
+---
+id: quarkus
+title: "Quarkus"
+---
+
+The [Quarkus](https://quarkus.io/) integration is a community extension that allows you to easily leverage Zeebe within your Quarkus environment.
+
+Essentially, Quarkus provides a wrapper around the [Zeebe Java Client](/apis-clients/java-client/index.md), and is preconfigured with sensible defaults so you can get started with minimal configuration. Add a dependency, implement a worker, and add your credentials in your Quarkus project.
+
+The integration provides also Quarkus dev services to start everything what you need as docker containers for local development. [Zeebe-dev-monitor](https://github.com/lorislab/zeebe-dev-monitor)  UI is also included as a dev service.
```

Comment:
I would adjust this to:
"The integration **also** provides ~~also~~ Quarkus **developer** services to start everything ~~what~~ you need as **D**ocker containers for local development. [Zeebe-dev-monitor](https://github.com/lorislab/zeebe-dev-monitor) **(remove extra space here)** UI is also included as a **developer** service.

Reply:
@christinaausley thank you for your review, I updated the change